### ISSUE TITLE
Loss become Nan after some time training

### ISSUE THREAD

**Issue by DrokBing** [hustvl/4DGaussians#35]
![Screenshot from 2023-11-18 11-32-03](https://github.com/hustvl/4DGaussians/assets/44105417/e1aed754-0c4f-438a-aa4f-a0d001c8adae)
And the rendering image is white background

**Comment by guanjunwu** [comment]
Wow... I also found same problem during optimization. Initialiy I think it's error on my training machine.
Most of cases happen on scenes have more background points such as `flame_salmon_1' and `coffee_martini` of the Neu3D datasets. I think it may be the nemerical overflow during training. Do you have any ideas?
I hope we can solve it together if you have time :)

**Comment by Arisilin** [comment]
I also encountered this problem when training on my own scene, the loss may become nan after several iterations in fine stage. Besides, there are also some cases that "Runtime Error: numel: integer multiplication overflow" happens during fine stage training. I am not sure if it is caused by similar reason.

**Comment by leo-frank** [comment]
I meet the same problem, on a colmap-format dataset. 

![image](https://github.com/hustvl/4DGaussians/assets/47302141/6f47fef8-7948-40d5-888d-61ad7267518c)

The PSNR suddenly decrease into an unexpected value(4.28), while the number of point cloud also decreases .

**Comment by guanjunwu** [comment]
I guess that maybe the scene's bounding box is so large, and causes the error when producing the backpropagation in the Gaussian deformation field network.

**Comment by GotFusion** [comment]
> I guess that maybe the scene's bounding box is so large, and causes the error when producing the backpropagation in the Gaussian deformation field network.

Is there any solution to solve this problem?

**Comment by zhaohaoyu376** [comment]
> In my test, set `no_dr=True` and `no_ds=True` (disable the deformation of rotation and scaling) will decrease the happening of the problem.

However, it seems that performance might be significantly affected by this approach. Are there any other solutions?

**Comment by zhaohaoyu376** [comment]
Why do I always run again because the loss is nan? I can't even finish running it once.

**Comment by shashanknsridhara** [comment]
Did anyone find a solution for this problem? I am using a custom dataset and I am encountering this error.
